lib/blacklight/component.rb

```diff
@@ -3,6 +3,12 @@
 module Blacklight
   class Component < ViewComponent::Base
     class << self
+      def inherited(subclass)
+        # Workaround for https://github.com/ViewComponent/view_component/issues/1565
+        subclass.config = ViewComponent::Base.config
```

Comment:
This obviously works, but it also means that you can't actually configure component classes in any way that varies from `Base.config`. This seems like it might not be desirable. The ViewComponent hooks initiailize to the `config.defaults`... also seems wrong in cases outside the Base class.

Reply:
should we dup it?